tests/testthat/test-basic.R

```diff
@@ -30,6 +30,17 @@ test_that("Connect", {
   DatabaseConnector::disconnect(connection)
 })
 
+test_that("Table names and column names case", {
+  connection <- DatabaseConnector::connect(getEunomiaConnectionDetails())
+  tableNames <- DatabaseConnector::getTableNames(connection)
```

Comment:
`tableNames <- DatabaseConnector::getTableNames(connection, cast = "none")`

Perhaps you should suppress the casting that is done by getTableNames to ensure the columns are actually lowercase in the database. I'd probably use DBI for this but  DatabaseConnector::getTableNames should work.

Reply:
@ablack3 I have updated the test